public/static/locales/fr/profile.json

```diff
@@ -33,8 +33,8 @@
       },
       "invalidInput": "Doit être au moins 4 caractères.",
       "unknownError": {
-        "title": "Unknown error[FR]",
-        "content": "An unknown error occurred and we could not process your request. Please try again later.[FR]"
+        "title": "Erreur inconnue",
+        "content": "Une erreur inconnue s'est produite et nous n'avons pas pu traiter votre demand. Veillez réessayer plus tard."
```

Comment:
`demande` instead of `demand`

Reply:
`Veuillez` instead of `Veillez`

Reply:
The rest looks good to me :)

Reply:
Merci!! :D J'ai fait ça trop rapidement évidemment 😅